fre/cmor/cmor_mixer.py

```diff
@@ -4,408 +4,383 @@
-        lon = ds.createVariable('lon', statics_lon.dtype, ('yh', 'xh') )
+        fre_logger.info('creating lat and lon variables in temp file')
+        lat = ds.createVariable('lat', statics_lat.dtype, ('yh', 'xh'))
+        lon = ds.createVariable('lon', statics_lon.dtype, ('yh', 'xh'))
         lat[:] = statics_lat[:]
         lon[:] = statics_lon[:]
 
-        fre_logger.info('\n')
+        fre_logger.info('')
         print_data_minmax(lat[:], "lat")
         print_data_minmax(lon[:], "lon")
-        fre_logger.info('\n')
-
+        fre_logger.info('')
 
         # grab the corners of the cells, should have shape (yh+1, xh+1)
-        fre_logger.info('reading geolat and geolon coordinates of cell corners from statics file \n')
-        lat_c = from_dis_gimme_dis(statics_ds,'geolat_c')
-        lon_c = from_dis_gimme_dis(statics_ds,'geolon_c')
+        fre_logger.info('reading geolat and geolon coordinates of cell corners from statics file')
+        lat_c = from_dis_gimme_dis(statics_ds, 'geolat_c')
+        lon_c = from_dis_gimme_dis(statics_ds, 'geolon_c')
 
-        fre_logger.info('\n')
+        fre_logger.info('')
         print_data_minmax(lat_c, "lat_c")
         print_data_minmax(lon_c, "lon_c")
-        fre_logger.info('\n')
-
+        fre_logger.info('')
 
         # vertex
-        fre_logger.info('creating vertex dimension\n')
+        fre_logger.info('creating vertex dimension')
         vertex = 4
         ds.createDimension('vertex', vertex)
 
-
         # lat and lon bnds
-        fre_logger.info('creating lat and lon bnds from geolat and geolon of corners\n')
-        lat_bnds = ds.createVariable('lat_bnds', lat_c.dtype, ('yh', 'xh', 'vertex') )
-        lat_bnds[:,:,0] = lat_c[1:,1:] # NE corner
-        lat_bnds[:,:,1] = lat_c[1:,:-1] # NW corner
-        lat_bnds[:,:,2] = lat_c[:-1,:-1] # SW corner
-        lat_bnds[:,:,3] = lat_c[:-1,1:] # SE corner
-
-        lon_bnds = ds.createVariable('lon_bnds', lon_c.dtype, ('yh', 'xh', 'vertex') )
-        lon_bnds[:,:,0] = lon_c[1:,1:] # NE corner
-        lon_bnds[:,:,1] = lon_c[1:,:-1] # NW corner
-        lon_bnds[:,:,2] = lon_c[:-1,:-1] # SW corner
-        lon_bnds[:,:,3] = lon_c[:-1,1:] # SE corner
-
-        fre_logger.info('\n')
+        fre_logger.info('creating lat and lon bnds from geolat and geolon of corners')
+        lat_bnds = ds.createVariable('lat_bnds', lat_c.dtype, ('yh', 'xh', 'vertex'))
+        lat_bnds[:, :, 0] = lat_c[1:, 1:]  # NE corner
+        lat_bnds[:, :, 1] = lat_c[1:, :-1]  # NW corner
+        lat_bnds[:, :, 2] = lat_c[:-1, :-1]  # SW corner
+        lat_bnds[:, :, 3] = lat_c[:-1, 1:]  # SE corner
+
+        lon_bnds = ds.createVariable('lon_bnds', lon_c.dtype, ('yh', 'xh', 'vertex'))
+        lon_bnds[:, :, 0] = lon_c[1:, 1:]  # NE corner
+        lon_bnds[:, :, 1] = lon_c[1:, :-1]  # NW corner
+        lon_bnds[:, :, 2] = lon_c[:-1, :-1]  # SW corner
+        lon_bnds[:, :, 3] = lon_c[:-1, 1:]  # SE corner
```

Comment:
This looks right, and we'll confirm by testing it on some CMIP6 ocean pp files and compare with the CMIP6 published ocean files.